src/DBALCompatibility.php

```diff
@@ -17,14 +17,19 @@ final class DBALCompatibility
 {
     public static function notSupportedPlatformException(string $method): Exception
     {
+        // phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName
         if (class_exists('Doctrine\DBAL\Platforms\Exception\NotSupported')) {
-            return NotSupported::new($method);
+            return \Doctrine\DBAL\Platforms\Exception\NotSupported::new($method);
```

Comment:
Why should this be inline?

Reply:
Because it is a backward compatibility class.
The complexity of supporting dbal3 and dbal4 requires this class to use fully qualified class names when dbal3 is used, because the class does not exist in dbal 3.